### ISSUE TITLE
App has been removed by Google play protect

### ISSUE THREAD

**Issue by sinancetinkaya** [SoulForeverInPeace/Boot-PC#20]
![Screenshot_20241110-121915_Google_Play_Store_1](https://github.com/user-attachments/assets/59a34a5c-f636-437f-8c13-c298a26353e0)
I tried to reinstall from here and it was blocked again
![Screenshot_20241110-123338_1](https://github.com/user-attachments/assets/e2022975-1899-4ad2-bae2-6d48c96ec734)

**Comment by Tanzin01** [comment]
I can voutch for it. Play store is lying.

This is the original app. Hence,  the1st sentence- "This app is fake", is a Lie . If possible , ask them to give the original app link since they are saying otherwise. 

The 2nd sentence- what a shameless liar play store and the people involved in it is.  Curse on them for lying. It's a grave lie .

The app is open source.  This app dosent contain any ** Internet ** connecting component /code . 
If you are a regular user, feel free to use a app to test if boot pc app can connect to Internet.

**Comment by GEOEGII555** [comment]
If there's no malware, then can you explain this?

https://github.com/SoulForeverInPeace/Boot-PC/commit/68a92402058adfef3113d49e4cbe88c6179a6a62

**Comment by GEOEGII555** [comment]
~~You say that there's no malware... Can you explain this commit? @Tanzin01~~

~~https://github.com/SoulForeverInPeace/Boot-PC/commit/68a92402058adfef3113d49e4cbe88c6179a6a62~~

~~Typical malware hiding attempt using spaces so it goes off screen.~~

![An image showing that the malicious code was "hidden" using spaces, but here it is still visible because of word wrap](https://github.com/user-attachments/assets/07991612-1da3-4dfb-aeac-9a6bafc46a0a)

Nevermind: https://github.com/SoulForeverInPeace/Boot-PC/issues/20#issuecomment-2669727765

**Comment by sinancetinkaya** [comment]
> You say that there's no malware... Can you explain this commit? [@Tanzin01](https://github.com/Tanzin01)
> 
> [68a9240](https://github.com/SoulForeverInPeace/Boot-PC/commit/68a92402058adfef3113d49e4cbe88c6179a6a62)
> 
> Typical malware hiding attempt using spaces so it goes off screen.
> 
> ![An image showing that the malicious code was "hidden" using spaces, but here it is still visible because of word wrap](https://github.com/user-attachments/assets/07991612-1da3-4dfb-aeac-9a6bafc46a0a)

That looks like a comment that belongs to an old commit

**Comment by GEOEGII555** [comment]
> > You say that there's no malware... Can you explain this commit? [@Tanzin01](https://github.com/Tanzin01)
> > [68a9240](https://github.com/SoulForeverInPeace/Boot-PC/commit/68a92402058adfef3113d49e4cbe88c6179a6a62)
> > Typical malware hiding attempt using spaces so it goes off screen.
> > ![An image showing that the malicious code was "hidden" using spaces, but here it is still visible because of word wrap](https://github.com/user-attachments/assets/07991612-1da3-4dfb-aeac-9a6bafc46a0a)
> 
> That looks like a comment that belongs to an old commit

Oh, I see now that it's after a comment, didn't notice it before. Why would it be there in the first place tho?

**Comment by sinancetinkaya** [comment]
@GEOEGII555 could be for previous tests, but I have no idea about choosing `rm -rf /*` for that...

**Comment by Tanzin01** [comment]
> You say that there's no malware... Can you explain this commit? [@Tanzin01](https://github.com/Tanzin01)
> 
> [68a9240](https://github.com/SoulForeverInPeace/Boot-PC/commit/68a92402058adfef3113d49e4cbe88c6179a6a62)
> 
> Typical malware hiding attempt using spaces so it goes off screen.
> 
> ![An image showing that the malicious code was "hidden" using spaces, but here it is still visible because of word wrap](https://github.com/user-attachments/assets/07991612-1da3-4dfb-aeac-9a6bafc46a0a)

How did you found that commit? Explain.

I had to waste 1hour or more. So so disgusted. I came to solve another issue (current first issue)  to support a13 devices if feasible. But this costed me lots of time  & efforts. Is that edited by you or something? mvm, probably not.

However, here is the thing,  as I said, I can vouch since I know the ins and outs of this project. I read the whole code. There was and is no such code like that including as comments.

If I visit  the commit given  by @GEOEGII555 , This notice is pinned at top,
`This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.`

![Image](https://github.com/user-attachments/assets/cd4c03a5-cffd-4da4-8fa6-add66d515fe9)

After searches , this notice could mean a lot is what I learnt. 

After wasting an hour, I still couldn't pin point this commit' source or whatever's source. And decided to let it go instead of wasting more of my time and effort. II'm not a big fan of git/github for reasons,  hence it could be someone else or something else or simply my lack of knowlegde. 

Anyway, check this out-
Notice the commit's date - `Jan 16 2023`

![Image](https://github.com/user-attachments/assets/7fd005d5-6770-40b4-a4d1-dabd0158fdc0)

And the most hilarious thing is,
@SoulForeverInPeace account creation date- `June 9 2023
`

![Image](https://github.com/user-attachments/assets/3b9ab63b-0625-4ec6-9b9d-f8d4944a3be9)

Basically , it's a senseless,  messed up, idk how to express this whole scenario. Simply speaking , a illogical, inconsistent situation.

also check the parent commit's date and @GEOEGII555's provided commits date. 

parent commit date -
`17 jan 2024`

![Image](https://github.com/user-attachments/assets/4b5bf7d1-6d3a-4ff1-bbde-8e6eb2842670)



Also, by the way , on a side note- it was somewhat a closed source project. He made it public later when few people started using this app. So , you know what, he wouldn't have made it public in the first place if he had  not so good intention.
